python/tvm/relay/op/contrib/cudnn.py

```diff
@@ -50,6 +51,8 @@ def partition_for_cudnn(
     tvm.IRModule
         The partitioned module.
     """
+    if params:
```

Comment:
Given it's a one liner never figured out why folks want to fold that into every partition function. Cargo culting?

Reply:
In my case, I want to pattern match against a mod where batch norm is removed by constant folding + fold scale axis. Param binding is a prereq for these passes.